backend/dataall/modules/worksheets/services/worksheet_permissions.py

```diff
@@ -47,3 +45,13 @@
 
 RESOURCES_ALL.append(RUN_ATHENA_QUERY)
 RESOURCES_ALL_WITH_DESC[RUN_ATHENA_QUERY] = 'Run Worksheet Athena queries on this environment'
+
```

Comment:
The code block from lines 48 to 57 is unnecessary. `DOWNLOAD_ATHENA_QUERY_RESULTS` is a permission that is applied on a Worksheet Resource.  `ENVIRONMENT_X` permissions always refer to permissions applied to Environment Resource. RESOURCES_ALL_WITH_DESC is already defined in line 32, but for Environment permissions such as `RUN_ATHENA_QUERY` in this case, we would want to display another description in the invite-team window.

Reply:
remove the unnecessary code

Reply:
Hi Anil, I think you forgot to push the changes :) there are also some conflicts to resolve. Could you merge main to your branch?